Xamarin.Forms.Platform.iOS/Renderers/TabbedRenderer.cs

```diff
@@ -387,6 +401,297 @@ void UpdateCurrentPage()
 			((TabbedPage)Element).CurrentPage = SelectedIndex >= 0 && SelectedIndex < count ? Tabbed.GetPageByIndex((int)SelectedIndex) : null;
 		}
 
+		void UpdateTabSwipes()
+		{
+			_tabSwipes = ((TabbedPage)Element).OnThisPlatform().TabSwipes();
+
+			if (_tabSwipes == null || _tabSwipes.Count == 0)
+			{
+				ShouldSelectViewController = null;
+
+				if (_uiPanGestureRecognizer == null)
+					return;
+
+				if(View.GestureRecognizers.Contains(_uiPanGestureRecognizer))
+					View.GestureRecognizers.Remove(_uiPanGestureRecognizer);
+
+				_uiPanGestureRecognizer.Dispose();
+				_uiPanGestureRecognizer = null;
```

Comment:
Removing and disposing `_uiPanGestureRecognizer` doesn't seem to be removing the target/handler (i.e. the handler gets called still when the object is disposed and set to null) Does anyone know why? GC should be relatively quick for cleanup. Not sure if this is a bug.